silverback/middlewares.py

```diff
@@ -7,24 +7,22 @@
 from taskiq import TaskiqMessage, TaskiqMiddleware, TaskiqResult
 
 from silverback.persistence import HandlerResult
-from silverback.types import SilverbackID, handler_id_block, handler_id_event
+from silverback.types import SilverbackID, TaskType, handler_id_block, handler_id_event
 from silverback.utils import hexbytes_dict
 
 
 def resolve_task(message: TaskiqMessage) -> Tuple[str, Optional[int], Optional[int]]:
-    block_number = None
-    log_index = None
+    block_number = message.labels.get("number") or message.labels.get("block")
+    log_index = message.labels.get("log_index")
     task_id = message.task_name
```

Comment:
though: this probably means nothing now.  Though it could be okay since this is primarily used in storing persistence data.  :shrug:

Reply:
yes, this was actually the primary reason I refactored, since I wanted the results to basically look like `user_task_name.result_A`